src/icon/__tests__/icon.test.tsx

```diff
@@ -39,10 +39,6 @@ describe('Icon Component', () => {
       const { container } = render(<Icon size="inherit" name="settings" />);
       expect(container.firstElementChild).toHaveClass(styles['icon-flex-height']);
     });
-    it('renders with inline style for height', () => {
-      const { container } = render(<Icon size="inherit" name="settings" />);
-      expect(container.firstElementChild).toHaveAttribute('style', 'height: 0px;');
```

Comment:
Can we keep this test to confirm that the parent size is reflected properly (even if it is zero in jsdom)?

Reply:
Fine with me, your call